<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileLatLongClass</class>
 <widget class="QDialog" name="TileLatLongClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TileLatLong</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Tile Number</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>X      </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_X">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Y      </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Y"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Level</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Level"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Tile Info</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Top     </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_ULLAT">
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Left     </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_ULLONG">
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Bottom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_RBLAT">
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Right   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_RBLONG">
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="lbl_derived">
           <property name="text">
            <string>Tile Width : </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Point Info</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Longitude</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Long">
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Latitude   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Lat">
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Level        </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_LLlevel"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Tile Number</string>
        </property>
        <widget class="QLabel" name="txtTileNum">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_Lat</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TileLatLongClass</receiver>
   <slot>Change_Lat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Long</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TileLatLongClass</receiver>
   <slot>Change_Long()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_LLlevel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TileLatLongClass</receiver>
   <slot>Change_LLlevel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Level</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TileLatLongClass</receiver>
   <slot>Change_Level()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Y</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TileLatLongClass</receiver>
   <slot>Change_Y()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_X</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TileLatLongClass</receiver>
   <slot>Change_X()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Change_Lat()</slot>
  <slot>Change_Long()</slot>
  <slot>Change_LLlevel()</slot>
  <slot>Change_Level()</slot>
  <slot>Change_Y()</slot>
  <slot>Change_X()</slot>
 </slots>
</ui>
